lms/static/scripts/frontend_apps/components/LMSGrader.js

```diff
@@ -8,6 +8,12 @@ import { getSidebarWindow } from '../../postmessage_json_rpc/server';
 import StudentSelector from './StudentSelector';
 import SubmitGradeForm from './SubmitGradeForm';
 
+/**
+ * @typedef User
+ * @prop {string} displayName
+ * @prop {string} userid
+ */
```

Comment:
This typedef was used elsewhere in the file but not defined.

Reply:
👍